src/index.html

```diff
@@ -7,6 +7,44 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div
+       class="calendar
+       calendar--start--day--sat
+       calendar--month--length--31"
+      >
```

Comment:
I think GitHub see the problem here because you have more then 1 class per block. I suggest you to do this if you don't want to see the Check failure on GitHub. But you don't have to do this.
```suggestion
       class="calendar">
```